DomainServices/Services/CustomerServices.cs

```diff
@@ -35,6 +36,7 @@ public void Add(Customer customer)
                 customer.Id = LastId + 1;
                 CustomersClients.Add(customer);
             }
+            return customer.Id;
         }
 
         public bool DeleteCustomer(int id)
```

Comment:
Nesse contexto estamos tratando unica e exclusivamente de Customer. Não é necessário a palavra Customer no método Delete.
```suggestion
        public bool Delete(int id)
```